models/prices/solana/prices_solana_tokens.sql

```diff
@@ -676,4 +676,5 @@ FROM
         ('scnsol-socean-staked-sol', 'solana',  'SCNSOL', '5oVNBeEEQvYi1cX3ir8Dx5n1P7pdxydbGF2X4TxVusJm', 9),
         ('cwif-catwifhat', 'solana', 'CWIF', '7atgF8KQo4wJrD5ATGX7t1V2zVvykPJbFfNeVf1icFv1', 2),
         ('sc-shark-cat', 'solana', 'SC', '6D7NaB2xsLd7cauWu1wKk6KBsJohJmP2qZH9GEfVi5Ui', 6)
+        ('bera-bera', 'solana', 'BERA', 'Hiqq4Kba2dawtgCtoysXKirk1KgBMSiT71FtMq76Aht', 6)
```

Comment:
missing comma on previous line

Reply:
looks like they do have the token, but it's inactive, you can see here:
https://api.coinpaprika.com/v1/coins/bera-bera

Reply:
i think you can request it to be activated here:
https://coinpaprika.com/add/

Reply:
Thanks @jeff-dude . It looks like they only support CEX tokens, because I am asked for a price API when filling the add token form.
Do you know if they support DEX tokens?

Reply:
hm, i'd imagine due to our large list of tokens across chains that DEX is for sure supported. can you skip that field? or maybe able to contact them and ask?

Reply:
Ok, I'll contact them. Thanks. Closing this PR for now and will reopen once I get a response